src/Components/programs/Program.js

```diff
@@ -15,23 +18,29 @@ export const Program = props => {
   const onPress = () => {
     const firstTodoTraining = trainings.find(training => !programCompleteTrainings.includes(training));
     if (firstTodoTraining !== undefined) {
```

Comment:
Do we still need that if and the fistToDoTraining computation ?

Reply:
yep you're right